recipes/pyod/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyod" %}
+{% set version = "0.7.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22cd28595187ca54e06418522b699b80ab6eae658ac3de194c3ef85730127016
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - combo
+    - joblib
+    - matplotlib
+    - numba >=0.35
+    - numpy >=1.13
+    - pip
+    - python
+    - scikit_learn >=0.19.1
+    - scipy >=0.19.1
+    - six
+    - suod
```

Comment:
Most of these are not needed to build the package and should be removed from host